doc/using_tf.rst

```diff
@@ -133,6 +133,26 @@ In your training script the channels will be stored in environment variables ``S
    ``output_path``.
 
 
+Use third-party libraries
+-------------------------
+
+If there are other packages you want to use with your script, you can use a ``requirements.txt`` to install other dependencies at runtime.
+
+Support for installing packages using ``requirements.txt`` varies with Tensorflow versions:
+
+- For Tensorflow 1.15.2 with Python 3.7 or newer, Tensorflow 2.2 or newer, and Script Mode without Horovod in any version:
+    - Include a ``requirements.txt`` file in the same directory as your training script.
+    - You must specify this directory using the ``source_dir`` argument when creating a Tensorflow estimator.
+- For older versions of Tensorflow using Script Mode with Horovod:
+    - Write a shell script for your entry point that first calls ``pip install -r requirements.txt``, then runs your training script.
```

Comment:
maybe link to [this example](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb) for demonstrating how to use a shell script?